services/movie.py

```diff
@@ -0,0 +1,40 @@
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> list[Movie]:
+
+    filters = {}
+
+    if genres_ids:
+        filters["genres__id__in"] = genres_ids
+    if actors_ids:
+        filters["actors__id__in"] = actors_ids
+
+    return Movie.objects.filter(**filters).distinct()
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
```

Comment:
Consider using `Movie.objects.filter(id=movie_id).first()` instead of `Movie.objects.get(id=movie_id)` to avoid `DoesNotExist` exceptions if the movie with the specified ID is not found. This will return `None` instead of raising an exception.